### ISSUE TITLE
Make draggable components icons more consistent

### ISSUE THREAD

**Issue by combinatorist** [gristlabs/grist-core#263]
**TL;DR:** There's a small inconsistency in the UI that originally caused me a little bit of confusion/friction. I thought it'd be easy to fix, but I got in a little over my head with the frameworks. But, I'm sure someone who knows this code base could take it from here to fix it quite quickly.

---

In the Page "treeview to the left", there's a double-vertical-bars icon to indicate where you can grab a "page" to reorder it in the list.

That appears to be set here: https://github.com/gristlabs/grist-core/blob/e73e74b8adcc256b2125ea3a94801059c4bbc523/app/client/ui/TreeViewComponent.ts#L333 referencing an icon which I believe is ultimately defined by an SVG.

You can see an example from one of my docs in the bottom right of this picture: 
<img width="192" alt="Screen Shot 2022-08-31 at 11 38 21" src="https://user-images.githubusercontent.com/10149067/187754711-4358f2a1-3cfd-4f16-bc07-f903cb1b7ea4.png">

Meanwhile, the widgets are dragged around by an icon to their top left, which look like 3 vertically-aligned dots. I believe this is defined as a character in the glyphicons font, although I can't seem to find the actual character when I inspect the page. But, I suspect it's set here: https://github.com/gristlabs/grist-core/blob/808aacdc522feeb11f74047e5bdaa5e58c49125a/app/client/components/ViewLayout.ts#L297

And you can see an example in the top right of this picture:
<img width="240" alt="Screen Shot 2022-08-31 at 11 41 52" src="https://user-images.githubusercontent.com/10149067/187755763-61ccd533-7813-4bde-a26f-b09b22a938e1.png">

---

My proposal is to use the first icon for both cases, since it has less chance of getting confused with the horizontal triple dots used as a dropdown icon to the top right of widgets.

**Comment by dsagal** [comment]
This would indeed be useful cleanup. There is even a TODO in [in ViewLayout.css](https://github.com/gristlabs/grist-core/blob/e73e74b8adcc256b2125ea3a94801059c4bbc523/app/client/components/ViewLayout.css#L44), which this would resolve.

The only difficulty is switching from the older style of code (which uses CSS classes and glyphicons) to the newer style (which uses grainjs styled elements and own icons), which is what tripped you up, @combinatorist, by the sound of it. If a nudge can help you over that trip-up, I suggest replacing this line you found https://github.com/gristlabs/grist-core/blob/808aacdc522feeb11f74047e5bdaa5e58c49125a/app/client/components/ViewLayout.ts#L297 with:

```
cssDragIcon('DragDrop',
```

and define `cssDragIcon` at the bottom of the same file, by styling `icon` with CSS moved from `.viewsection_drag_indicator` in `ViewLayout.css`:
```
const cssDragIcon = styled(icon, `
  visibility: hidden;
  --icon-color: ${colors.slate};
  /* ...other icon styles if needed... */
  
  .viewsection_title:hover &.layout_grabbable {
    visibility: visible;
  }
`);
```

If you try this and run into any issues, let us know -- it should not be far from working...

**Comment by combinatorist** [comment]
Oh, wow - thanks for the tip, @dsagal! I only just noticed your response, but I'll try your suggestion out soon - hopefully this week!

**Comment by yohanboniface** [comment]
hi @combinatorist are you working on a fix or should we propose something ? :)

**Comment by yohanboniface** [comment]
cc @LouisDelbosc ? Sounds quick too :)

**Comment by tayflo** [comment]
This issue seems now solved to me and could be closed.
